Makefile

```diff
@@ -317,6 +320,32 @@ bundle-base: manifests kustomize operator-sdk ## Generate bundle manifests and m
 
 export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
 
+.PHONY: add-console-plugin-annotation
+add-console-plugin-annotation: ## Add console-plugin annotation to the CSV
+	# add console-plugin annotation
+	$(YQ) -i '.metadata.annotations."console.openshift.io/plugins" = "[\"node-remediation-console-plugin\"]"' ${CSV}
+
+.PHONY: add-replaces-field
+add-replaces-field: ## Add replaces field to the CSV
+	# add replaces field when building versioned bundle
+	@if [ $(VERSION) != $(DEFAULT_VERSION) ]; then \
+		if [ $(PREVIOUS_VERSION) == $(DEFAULT_VERSION) ]; then \
```

Comment:
Good catch, it does not prevent the two values to be equal.

However this is a check for preventing PREVIOUS_VERSION not to have a value set (unless the special case of 0.0.1).

A complete check should prevent not only the two values to be the same, but also VERSION to be higher than PREVIOUS_VERSION.
Since this is the same check done in all operators' Makefiles at the moment, I would experiment different checks in a follow up PR

Reply:
>VERSION to be higher than PREVIOUS_VERSION.

Correct
>Since this is the same check done in all operators' Makefiles at the moment, I would experiment different checks in a follow up PR

Ok, it is something that we should care about